src/SPC/builder/linux/LinuxBuilder.php

```diff
@@ -29,12 +29,14 @@ public function __construct(array $options = [])
         // check musl-cross make installed if we use musl-cross-make
         $arch = arch2gnu(php_uname('m'));
 
-        // set library path, some libraries need it. (We cannot use `putenv` here, because cmake will be confused)
-        $this->setOptionIfNotExist('library_path', "LIBRARY_PATH=/usr/local/musl/{$arch}-linux-musl/lib");
-        $this->setOptionIfNotExist('ld_library_path', "LD_LIBRARY_PATH=/usr/local/musl/{$arch}-linux-musl/lib");
-
         GlobalEnvManager::init($this);
 
+        // set library path, some libraries need it. (We cannot use `putenv` here, because cmake will be confused)
+        if (getenv('SPC_NO_MUSL_PATH') !== '1') {
```

Comment:
Use `yes` instead?

Reply:
the env reader interprets "yes" or "on" as "1".

Reply:
At least when the env file is parsed. I'm not sure about the case when it's set as an environment variable.

If it behaves differently there and passes "yes", we may have to compare in_array($var, ["1", "yes", "on", "true"]).

Reply:
Currently getenv doesn't do the conversion, so a better solution might be `filter_var`.

Reply:
I'll push it later, sorry for not considering it earlier